packages/sitebuilder/src/constants/index.ts

```diff
@@ -6,8 +6,7 @@ export const ASSETS_URL = SITEBUILDER.assets_url;
 
 export const IMAGE_DIR = `${ ASSETS_URL }images/`;
 
-// export const CARDS: SetupCardInterface[] = SITEBUILDER.cards || [];
-export const CARDS: SetupCardInterface[] = cardsJson;
+export const CARDS: SetupCardInterface[] = SITEBUILDER.cards || [];
```

Comment:
You probably want to remove the `cards.json` import and the entire file unless you're holding onto it for some reason.